scripts/moduleReport.ts

```diff
@@ -256,15 +301,33 @@ async function checkBaseRealtimeFiles() {
 }
 
 (async function run() {
-  const errors: Error[] = [];
-
-  errors.push(...printAndCheckMinimalUsefulRealtimeBundleSize());
-  errors.push(...printAndCheckModuleSizes());
-  errors.push(...printAndCheckFunctionSizes());
-  errors.push(...(await checkBaseRealtimeFiles()));
+  const output = (
+    await Promise.all([
+      calculateAndCheckMinimalUsefulRealtimeBundleSize(),
+      calculateAllModulesBundleSize().then((tableRow): Output => ({ tableRows: [tableRow], errors: [] })),
```

Comment:
let's make `calculateAllModulesBundleSize` return `Output` object directly and remove this .then call for consistency with other calculate functions